src/main/resources/schema/provider.definition.schema.v1.json

```
@@ -301,6 +301,11 @@
             ],
             "$ref": "#/definitions/httpsUrl"
         },
+        "taggable": {
+            "$comment": "A boolean flag indicating whether this resource type supports updatable tagging.",
+            "type": "boolean",
+            "default": "true"
```

@Yunhao-Jiao had a good catch that we know `taggable` is `false` if an `UpdateHandler` isn't present
```suggestion
            "type": "boolean"
```
think it might make more sense to default to `true` if there's an `UpdateHandler`, and `false` if not

---

especially since only a few of the [existing 38 AWS resource providers without `UpdateHandler`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-type-schemas.html) set `taggable`:
```bash
CloudformationSchema $ grep -L '"update" : {' * | wc -l # resource schemas without "update"
      38
CloudformationSchema $ grep -L '"update" : {' * | xargs grep 'taggable'
aws-backup-backupselection.json:  "taggable" : false,
aws-route53-dnssec.json:  "taggable" : false,
aws-servicecatalogappregistry-attributegroupassociation.json:  "taggable" : false,
aws-servicecatalogappregistry-resourceassociation.json:  "taggable" : false,
aws-ses-configurationset.json:  "taggable" : false,
```

---

https://github.com/aws-cloudformation/cloudformation-resource-schema/pull/122

---

the whole issue with lack of update handler is a CFN specific issue, and that is orthogonal to the issue here.  A resource can be taggable (i.e., it supports tags) and also not have an update handler.  Taggable is true by default because we expect that all resources should support tags

---

I honestly think `taggable` is too nuanced to be adequately captured by a `boolean`:
https://github.com/aws-cloudformation/cloudformation-resource-schema/pull/105#issuecomment-827975377
but assuming we stick with the definition introduced in this PR, which is that `taggable` means updatable tags, https://github.com/aws-cloudformation/cloudformation-resource-schema/pull/122 seems necessary to me